pyface/qt/QtWebKit.py

```diff
@@ -27,6 +29,7 @@
             QWebEngineSettings as QWebSettings,
```

Comment:
Q : Is there a reason why we can't import QWebEngineHistory from PySide2 because we import it from PyQt5.

Reply:
I haven't investigated fully, but I recall that PySide2 didn't provide `QWebEngineHistory` at least at some point.  Unfortunately PyQt and PySide don't always have completely compatible APIs.  I'll do a quick check to see if it is in the current PySide2 release.

Reply:
Looks like it now works.